### PR TITLE
added a new pathway peepul building

### PR DESCRIPTION
**Pull request by Shivansh3218**
Adding a new pathway peepul building

### CODE REVIEW

File: src/components/PathwayCourse/index.js [hunk 1]
```diff
@@ -147,6 +155,9 @@ function PathwayCourse() {
     return state;
   });
 
+  console.log(pathwayId, "pathway id current")
+  console.log(completedAll, completedPortion, "completed portion")
```
Comment (Poonam-Singh-Bagh) [inline src/components/PathwayCourse/index.js]: @Shivansh3218 Remove console statements

File: src/components/Header/DropDown.js [hunk 2]
```diff
@@ -31,7 +32,7 @@ import {
 } from "@mui/material";
 
 const students = {
-  image: [python, typing, language, web, residential, random, amzbootcamp],
+  image: [python, typing, language, web, residential, random, peepul],
```
Comment (Poonam-Singh-Bagh) [inline src/components/Header/DropDown.js:35]: @Shivansh3218 If you're using `peepul.png` only then remove the other 2 images

File: src/components/PathwayCourse/customSnackbar.js [hunk 3]
```diff
@@ -4,15 +4,24 @@ import theme from "../../theme/theme";
 import Message from "../../components/common/Message/index";
 
 const CustomSnackbar = ({ openSnackbar, handleSnackbar, pathwayName }) => {
+  console.log(pathwayName);
```
Comment (Poonam-Singh-Bagh) [inline src/components/PathwayCourse/customSnackbar.js]: Remove console statement

**Review by Poonam-Singh-Bagh — CHANGES_REQUESTED** [review]

**Comment by Poonam-Singh-Bagh** [comment]
@Shivansh3218 resolve the conflicts carefully

File: src/msg/en.js [hunk 4]
```diff
@@ -29,5 +29,7 @@ en.LOGOUT = "Logout";
 en.TUTOR = "Tutor";
 en.CERTIFICATE_COMPLETION_WARNING =
   "Please complete all the courses to unlock %1 certificate";
+  en.PEEPUL_CERTIFICATE_COMPLETE_WARNING =
+  "Please complete at least 80% of course to unlock %1 certificate";
```
Comment (Poonam-Singh-Bagh) [inline src/msg/en.js:33]: Why is it ` %1` certificate?

**Review by Poonam-Singh-Bagh — CHANGES_REQUESTED** [review]

File: src/msg/en.js (hunk 4 above)
Comment (Shivansh3218) [reply src/msg/en.js:33]: %1 is for adding the name of pathway course for which the certificate needs to be provided

**Pull request merged**